### PR TITLE
new url link

### PR DESCRIPTION
**Pull request by renews**
This is the updated url, direct from the site.

### CODE REVIEW

File: Casks/franz.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
   sha256 :no_check
 
-  # www.dropbox.com/s/daepg6hdvaavfgv was verified as official when first introduced to the cask
-  url 'https://www.dropbox.com/s/daepg6hdvaavfgv/Franz.dmg?dl=1'
+  #This bit.ly link comes from the official site 
+  url 'http://bit.ly/22yh2b6'
   name 'Franz'
   homepage 'http://meetfranz.com'
-  license :gratis
+  license :free
```
Comment (vitorgalvao) [inline Casks/franz.rb]: This is wrong. `:free` isn’t a valid license type, `license` isn’t freeform. `:gratis` is correct.

File: Casks/franz.rb [hunk 2]
```diff
@@ -2,11 +2,11 @@
   version :latest
   sha256 :no_check
 
-  # www.dropbox.com/s/daepg6hdvaavfgv was verified as official when first introduced to the cask
-  url 'https://www.dropbox.com/s/daepg6hdvaavfgv/Franz.dmg?dl=1'
+  #This bit.ly link comes from the official site
```
Comment (vitorgalvao) [inline Casks/franz.rb]: Comment is non-standard, it needs to follow the same pattern as the previous one.

File: Casks/franz.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 cask 'franz' do
   version :latest
   sha256 :no_check
-
-  # www.dropbox.com/s/daepg6hdvaavfgv was verified as official when first introduced to the cask
-  url 'https://www.dropbox.com/s/daepg6hdvaavfgv/Franz.dmg?dl=1'
+  
+  # http://bit.ly/22yh2b6 was verified as official from the official site
```
Comment (vitorgalvao) [inline Casks/franz.rb]: No, it needs to be *precisely* `# http://bit.ly/22yh2b6 was verified as official when first introduced to the cask`. We need consistency in these comments.

File: Casks/franz.rb [hunk 4]
```diff
@@ -1,9 +1,9 @@
 cask 'franz' do
   version :latest
   sha256 :no_check
-
-  # www.dropbox.com/s/daepg6hdvaavfgv was verified as official when first introduced to the cask
-  url 'https://www.dropbox.com/s/daepg6hdvaavfgv/Franz.dmg?dl=1'
+
```
Comment (vitorgalvao) [inline Casks/franz.rb]: Trailing whitespace needs to be removed.

**Comment by vitorgalvao** [comment]
+ Please [amend](http://stackoverflow.com/a/179147/1661012) the commit message to include the name of the updated cask.

**Comment by renews** [comment]
i give up.

**Comment by vitorgalvao** [comment]
[Simple change](https://github.com/caskroom/homebrew-cask/pull/20092/files).

**Pull request closed without merging**